src/vendor/Path.ts

```diff
@@ -69,7 +85,7 @@ export default class Path {
     */
   readlink(): Path {
     try {
-      const output = Deno.readLinkSync(this.string)
+      const output = Deno.build.os == "windows" ? Deno.realPathSync(this.string) : Deno.readLinkSync(this.string)
```

Comment:
please add comments explaining why unexpected things are necessary. To me, this seems wrong.